alerts/charts/templates/ca-secret-issuer-cert.yaml

```diff
@@ -1,5 +1,7 @@
 {{- if and .Values.alerts.alertmanager.enabled .Values.alerts.alertmanager.ingress.enabled }}
-{{- if not (lookup "v1" "Secret" $.Release.Namespace (printf "%s-%s" $.Release.Namespace "monitoring-ca")) }}
+{{- $caSecret := (lookup "v1" "Secret" $.Release.Namespace (printf "%s-%s" $.Release.Namespace "monitoring-ca")) }}
+{{- if not $caSecret }}
+{{- $ca := genCA (printf "%s-%s" $.Release.Name "custom-ca") 3650 }}
```

Comment:
Question: Can the certificate be valid for 10 years? 🤔

Reply:
Thanks for the question. This is actually just a random high value. We came up with this automatically generated value to start somewhere. I will include this in an issue so as not to forget it.